src/middleware/flash_message.ml

```diff
@@ -0,0 +1,56 @@
+module Dream = Dream__pure.Inmost
+
+
+let log =
+  Log.sub_log "dream.flash_message"
+
+
+let five_minutes = 5. *. 60.
+
+
+let storage = Dream.new_local ~name:"dream.flash_message" ()
+
+
+let flash_cookie = "dream.flash_message"
+
+
+let flash_messages inner_handler request =
+  let outbox = ref [] in
+  let request = Dream.with_local storage outbox request in
+  let%lwt response = inner_handler request in
+  Lwt.return(
+    let entries = List.rev !outbox in
+    let content = List.fold_right (fun (x,y) a -> `String x :: `String y :: a) entries [] in
+    let value = `List content |> Yojson.Basic.to_string in
```

Comment:
I'm not sure if using Yojson to serialize is the right approach.

Reply:
It is *a* right approach.